pkg/client/v1/nimbleos/ns_n_i_c.go

```diff
@@ -1,18 +1,17 @@
 // Copyright 2020 Hewlett Packard Enterprise Development LP
 
-package model
-
+package nimbleos
 
 // NsNIC - Network interface controller.
 // Export NsNICFields for advance operations like search filter etc.
 var NsNICFields *NsNIC
 
-func init(){
-	Namefield:= "name"
-	SubnetLabelfield:= "subnet_label"
-	DataIpfield:= "data_ip"
-		
-	NsNICFields= &NsNIC{
+func init() {
+	Namefield := "name"
+	SubnetLabelfield := "subnet_label"
+	DataIpfield := "data_ip"
+
+	NsNICFields = &NsNIC{
```

Comment:
When creating file names, I guess the generator adds an underscore before all capital letters.  It has some odd results like NsNic becoming ns_n_i_c when ns_nic would be better.  Other examples are ID and IDKV.  Perhaps we could suppress the underscore between consecutive capital letters if that does cause an incorrect result elsewhere.  Otherwise we could teach it to handle particular acronyms.

Reply:
Sure we can do that.